registry_quickstart/administrators/system_configuration.adoc

```diff
@@ -128,6 +131,16 @@ Restart the origin service to update the running configuration.
 $ sudo docker restart origin
 ----
 
+== Using Service Account Tokens for Authentication
+
+Typically long-lived, token-based authentication is desired. As an alternative
+to using user session tokens that expire, users may use
+link:../../admin_guide/service_accounts.html[service account tokens] to
```

Comment:
We now use xrefs instead of links. This should now be:
xref:../../admin_guide/service_accounts.adoc[service account tokens]